extensions/standard-processors/tests/integration/TLSServerSocketSupportedProtocolsTest.cpp

```diff
@@ -151,10 +151,12 @@ class SimpleSSLTestClient  {
   gsl::not_null<std::shared_ptr<logging::Logger>> logger_{gsl::make_not_null(logging::LoggerFactory<SimpleSSLTestClient>::getLogger())};
 
   static SocketDescriptor openConnection(const char *host_name, const char *port, logging::Logger& logger) {
-    struct addrinfo hints = {0}, *addrs;
+    struct addrinfo hints;
+    hints.ai_flags = 0;
```

Comment:
Is this the same behaviour? What was the issue with the `{0}` initialization?
https://man7.org/linux/man-pages/man3/getaddrinfo.3.html

Reply:
You are right it wasn't the same behavior and it failed on VS2019 build. I fixed it in [a89e77](https://github.com/apache/nifi-minifi-cpp/pull/1020/commits/a89e77484a6ea3651a2952a6a9236ea0e7027e17). The warning I got from this is that it did not initialize all members with the `{0}` initialization.